### DISCUSSION TITLE
Pushed my branch up and merged, but all my changed have disappeared

### DISCUSSION THREAD

**Discussion by Gabrielle-Tobermann** [nss-evening-cohort-14/evening-client-side#165] [Help Me!] [answered]
Gabby Tobermann 

Other help tickets: no 

Issue: I just finished creating my boards. I ran git add -A and committed my changed, then pushed my create-boards branch. Then I went to github to merge that into development branch. It said it had successfully merged so I went to my local and pulled down development. Now, I've lost all my changes locally and on Github for my create-boards branch. 

What I have tried: 
- git log: my commit is there 
- looking into development on github, my changed are NOT there. 
- checked into my create-boards branch, my changes are not there either. 
- tried remaking a new branch and add the form. When I run git status, it says there is nothing to commit???

Google links: 
https://git-scm.com/book/en/v2/Git-Basics-Viewing-the-Commit-History
https://stackoverflow.com/questions/4454621/git-changes-being-lost-why
http://gitready.com/advanced/2009/01/17/restoring-lost-commits.html

Links to PR: 
https://github.com/Gabrielle-Tobermann/ASSIGNMENT-Firebase-Pinterest/pull/22

**Comment by drteresavasquez** [comment]
@Gabrielle-Tobermann is this the commit? https://github.com/Gabrielle-Tobermann/ASSIGNMENT-Firebase-Pinterest/commit/f538a3f16e1b79a433ac02469358fcb79e0cf00e

**Reply by drteresavasquez** [reply]
Reminder to come back to this when we hit react

**Comment by Gabrielle-Tobermann** [comment, marked as answer]
Ubuntu doesn't like folder/file names to be capitalized or have numbers in them. They must be lowercase (with camel casing if needed).
